### PR TITLE
allow for building 'all' artifacts serially when not on the cluster

### PR DESCRIPTION
**Pull request by KjellSwedin**

### CODE REVIEW

File: {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
 
 """
-from pathlib import Path
 import shutil
 import sys
 import time
-from typing import Union
-
 import click
+
```
Comment (rmudambi) [inline {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py:13]: Why did you move these import statements down?

File: {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py [hunk 2]
```diff
@@ -69,8 +81,13 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
...
                 path.unlink()
 
-        build_all_artifacts(output_dir, verbose)
-
+        if running_from_cluster():
+            # parallel build when on cluster
+            build_all_artifacts(output_dir, verbose)
+        else:
+            # serial build when not on cluster
+            for loc in project_globals.LOCATIONS:
```
Comment (rmudambi) [inline {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py:89]: This will prompt the user before each location if there is an existing artifact at the location. It would be better either to prompt only once or prompt once for each location, but to do them all up front.

File: {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py (hunk 1 above)
Comment (KjellSwedin) [reply {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py:13]: Trying to get closer to this:
https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

File: {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py (hunk 2 above)
Comment (KjellSwedin) [reply {{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py:87]: Yes, I stepped back and reimplemented things. See if you think is is better.
